apis/datadoghq/v2alpha1/datadogagent_types.go

```diff
@@ -221,6 +231,10 @@ type LiveProcessCollectionFeatureConfig struct {
 	// Default: false
 	// +optional
 	StripProcessArguments *bool `json:"stripProcessArguments,omitempty"`
+
+	// RunInCoreAgent controls enablement of running the feature in the core agent.
+	// +optional
+	RunInCoreAgent *ProcessesRunInCoreAgent `json:"runInCoreAgent,omitempty"`
```

Comment:
It may be better to have one top level configuration for this to avoid having to handle inconsistent values as it adds a lot of logic. Is there a good way to do that?